src/UI/Component/Panel/Sub.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel;
+
+/**
+ * This describes a Sub Panel.
+ */
+interface Sub extends Panel {
```

Comment:
You could introduce another interface here that bundles the content-part of Sub and Standard.